tools/webpack-make

```diff
@@ -190,7 +190,7 @@ function generateDeps(makefile, stats) {
     lines.push("\trm -rf $(" + prefix + "_OUTPUTS) $(" + prefix + "_INSTALL) " + makefile);
     lines.push("all-local:: " + prefix);
     lines.push("\t@true");
-    lines.push("clean-local:: clean-" + prefix);
+    lines.push("distclean-local:: clean-" + prefix);
```

Comment:
Well, I can remove that patch again (it's not important enough to waste a lot of time on it). I thought this was supposed to help people who build/install it from the releases, but if we didn't get some complaints there, then we don't need the maintainer-mode stuff either IMHO.